renovate.json5

```diff
@@ -14,6 +14,20 @@
         "core/package.json"
       ]
     },
+    {
+      matchPackageNames: ["@stencil/react-output-target"],
+      matchFileNames: [
+        "core/package.json"
+      ],
+      allowedVersions: ["<=0.5.3"]
+    },
+    {
+      matchPackageNames: ["@stencil/core"],
+      matchFileNames: [
+        "core/package.json"
+      ],
+      allowedVersions: ["<=4.20.0"]
+    },
```

Comment:
I _think_ this will work to prevent these packages from updating, but hard to know until it merges and we let Renovate run 🤷‍♂️

Reply:
I'm also gonna create tasks for these respective issues so I'll add a comment with each of those task numbers once I have them